test-example-diagrams/src/main/resources/multiinstances-service-task/MultiInstancesServiceTask.md

```diff
@@ -0,0 +1,27 @@
+# Multi Instance Service Task
```

Comment:
Also here, loop cardinalities are not possible at the moment in C8.

Reply:
I split in 2 test games